src/Home/index.md

```diff
@@ -4,9 +4,9 @@
 
 ## Welcome to the documentation of the Vinegar project!
 
-- Vinegar is a transparent, minimal, configurable, fast bootstrapper and launcher for Roblox Player and Roblox Studio.
+- Vinegar is a transparent, minimal, configurable, and fast bootstrapper and launcher for Roblox Player and Roblox Studio.
```

Comment:
This doesnt seem right, theres two ands now, it would make sense if it was `fast bootstrapper for` simply, but its both a bootstrapper and a launcher simutaniously. Alternatively the description can be changed to something else that can make sense when vinegar is both a bootstrapper and a launcher

Reply:
two ands is technically correct, but it is oddly worded

Reply:
bootstrapper itself implies launcher, so we are good

Reply:
then remove launcher